### PR TITLE
Skip missing files

### LINKED ISSUE #26: Don't error on missing files
I don't think it should error when the files you specified doesn't exists. It should log it, but exit with error code `0`. Currently you have to use `trash --force` if you want to use it for cleaning up a build dist dir or something, as the directory might not exists.

So need to differentiate between missing files and actual errors. Should be easy. Just use [`path-exists`](https://github.com/sindresorhus/path-exists) on them. I know this is a race condition, but shouldn't matter and probably easier than checking afterwards.

Thoughts?

-

//@kevva @arthurvr

### PR DESCRIPTION
**Pull request by kevva**
Was going to add a test, but I'm in a hurry so it'll have to wait :).

Fixes #26.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,8 +39,10 @@
     "script"
   ],
   "dependencies": {
+    "each-async": "^1.1.1",
     "meow": "^3.0.0",
     "osx-trash": "^1.0.0",
+    "path-exists": "^1.0.0",
```
Comment (arthurvr) [inline package.json]: This was already in the `devDependencies` too. Let's remove it there.

**Comment by arthurvr** [comment]
The appveyor failure seems unrelated.

**Comment by sindresorhus** [comment]
Lgtm, but needs a test as you said.

File: package.json (hunk 1 above)
Comment (sindresorhus) [inline package.json]: I prefer to duplicate deps. If I suddenly don't need it in the CLI and remove it, tests will fail. Like everything I would prefer for deps to be explicit.

Comment (arthurvr) [inline package.json]: Oh, alright.

**Comment by kevva** [comment]
Yup, I'll write one.

**Comment by kevva** [comment]
@sindresorhus, added test.

**Comment by arthurvr** [comment]
LGTM - nice work @kevva!

**Comment by sindresorhus** [comment]
Superb :)

**Pull request merged**